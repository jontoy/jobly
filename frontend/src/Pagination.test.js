import React from 'react';
import { render } from '@testing-library/react';
import Pagination from './Pagination';

it('renders without crashing', () => {
    render(<Pagination />);
});
  
it('matches snapshot when user is logged in', () => {
      const {asFragment} = render(<Pagination />);
    expect(asFragment()).toMatchSnapshot();
});
