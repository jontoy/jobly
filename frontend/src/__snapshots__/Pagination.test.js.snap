// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot when user is logged in 1`] = `
<DocumentFragment>
  <nav
    aria-label="Pagination"
    class="Pagination"
  >
    <ul
      class="pagination"
    >
      <li
        class="page-item"
      >
        <button
          class="page-link"
        >
          Previous
        </button>
      </li>
      <li
        class="page-item"
      >
        <button
          class="page-link"
        />
      </li>
      <li
        class="page-item"
      >
        <button
          class="page-link"
        >
          Next
        </button>
      </li>
    </ul>
  </nav>
</DocumentFragment>
`;
